src/pages/ServiceDetails/ServiceInfo.tsx

```diff
@@ -130,11 +140,11 @@ class ServiceInfo extends React.Component<ServiceDetails, ServiceInfoState> {
                     <NavItem eventKey={2}>{'Source Services (' + Object.keys(dependencies).length + ')'}</NavItem>
                     <NavItem eventKey={3}>
                       {'Virtual Services (' + virtualServices.length + ')'}
-                      {validationChecks.hasVirtualServiceChecks ? errorIcon : undefined}
+                      {validationChecks.hasVirtualServiceChecks ? getValidationIcon() : undefined}
                     </NavItem>
                     <NavItem eventKey={4}>
                       {'Destination Rules (' + destinationRules.length + ')'}
-                      {validationChecks.hasDestinationRuleChecks ? errorIcon : undefined}
+                      {validationChecks.hasDestinationRuleChecks ? getSeverityIcon() : undefined}
```

Comment:
DestinationRules might have warnings as well.
Not today, but in next step, so I think this logic should be generic for both.

Reply:
So should we prepare the code or we should wait until next update?

Reply:
Instead to have a getValidationIcon() and getSeverityIcon() for the same task.
I would unify it now to make it generic.
Indeed, previous logic was generic, but it didn't consider 'warning' case.

Reply:
why are you using `getValidationIcon()` for virtual services and `getSeverityIcon()` for DestinationRules? what is the purpose?